proposed/http-middleware/middleware.md

```diff
@@ -31,12 +31,11 @@ and the creation of a resulting HTTP Response, as defined by PSR-7.
 
 Middleware using this standard MUST implement the following interface:
 
-- `Psr\Http\ServerMiddleware\MiddlewareInterface`
+- `Psr\Http\Server\ServerMiddlewareInterface`
 
-Middleware dispatching systems using this standard MUST implement the following
-interface:
+Handlers using this standard MUST implement the following interface:
```

Comment:
Isn't that implied by the fact that an implementation of an interface method cannot modify the method signature (even with parameter type widening)?